js/forum/dist/extension.js

```diff
@@ -62,6 +63,27 @@ System.register("wuethrich44/sso/main", ["flarum/extend", "flarum/app", "flarum/
                     items.remove('changeEmail');
                     items.remove('changePassword');
                 }
+
+                function checkRemoveAccountSection(items) {
+                    if (items.has('account')) {
```

Comment:
I submitted a PR https://github.com/flarum/core/pull/1218 to add the `isEmpty()` utility to ItemList. I added the part to check if it is implemented to make sure it is compatible with the current flarum beta7 release. After the PR is added into flarum's core release, the emulation part can be removed. Plus, `ItemList` does not have `props` attribute. My current implementation is the best way I know to [check if an object is empty]( https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object)